tests/dummy/app/templates/docs/motions.md

```diff
@@ -0,0 +1,61 @@
+# Motions
+
+A motion animates a single sprite. Each motion takes a sprite as the first argument and options as the second argument. Motions can also have `to` and `from` arguments. These arguments take initial and final attribute values of the sprite. Motions animate the change of the given attribute from its initial to its final value. Like transitions, you may use any of the built in motions or create your own. 
+
+
+### move
+Animates a sprite from its current position to its final position using css transforms. The only option argument it takes is `easing`. Because this motion works with `keptSprites`, it requires the start point and the end point of a sprite. 
+
+{{#docs-demo as |demo|}}
+    {{#demo.example name="move"}}
+      {{transitions-example}}
+    {{/demo.example}}
+
+    {{demo.snippet 'transitions-snippet.hbs' label='transitions-example.hbs'}}
+    {{demo.snippet 'transitions-snippet.js' label='transitions-example.js'}}
+    {{demo.snippet 'transitions-snippet.css'}}
+{{/docs-demo}}
+
+### moveSVG
+Animates one property on a svg element. This motion takes in an attribute name and animates that attribute from its initial to its final value. For example, on a circle `moveSVG` animates the `cx` property.
+
+{{#docs-demo as |demo|}}
+    {{#demo.example name="svg"}}
+        {{svg-example}}
+    {{/demo.example}}
+
+    {{demo.snippet 'svg-snippet.hbs' label='svg-example.hbs'}}
+    {{demo.snippet 'svg-snippet.js' label='svg-example.js'}}
+{{/docs-demo}}
+
+
+### compensateForScale
+Animates a sprite moving into or out of an area of a page that has scaling applied.
+
+### adjustColor
+Animates the change in color of a sprite. The `from` argument takes a css color, otherwise it defaults to the `measuredInitial` color and animates to the default final color. 
```

Comment:
This needs some more specifics. Is a "css color" a hex value as a string, like `#FFF`? or `#FFFFFF`? or `rgb(0 0 0)`? `white`? If any of those are valid, it should just say "any value that would be valid for the a CSS `color` attribute, such as xyz.